### DISCUSSION TITLE
Webrtc data channel

### DISCUSSION THREAD

**Discussion by randydom** [MagicFoundation/Alcinoe#348] [Q&A]
Hello, any hint or sample code on how:
Send message ( text ) between 2 peers using data channel please.

Thank you in advance.

**Comment by Zeus64** [comment]
Hi, did you try the ALNotificationService demo ?

**Reply by randydom** [reply]
But there's no WebRtc data channel use/ref in that demo ! 
Or did i miss something ?

Thank you

**Comment by Zeus64** [comment]
Oh, I see! You were asking about WebRTC. I thought you were referring to Firebase Data Messaging. Sorry, I don’t have any WebRTC-specific samples for that.

By the way, we’re currently looking for someone to help us migrate our WebRTC wrapper to the latest version. If you're interested in working with WebRTC, let us know!
